projects/apps/ChickCoop.py

```diff
@@ -10,11 +10,12 @@
     analytics_key=None,
     metrics=[
         SmartContractInteraction(
-            "Shop/Buying Gem and Lucky Wheel",
+            "Shop/Buying Gem, Lucky Wheel and Check-in",
             addresses=[
                 "EQCkc4zcv43qW8_0D-FD3Br-fxvA2IoQHCSg0o3cguyQ8Nh9",
                 "EQAOsRZKTrMe8ZTtwxxrAC8P84zgmYMF5YML0_NpL0jDrJRq",
-                "EQBUwiwJ5Wczc5QgOo6MKL-XwSaC9z0a46abdLWqRKb5yUt6"
+                "EQBUwiwJ5Wczc5QgOo6MKL-XwSaC9z0a46abdLWqRKb5yUt6",
+                "UQDd29ae4JYnoWAo9eFC4B_PJ0_eYoGaG1KAC3F-So-zJE3B"
```

Comment:
please use Bounceable format for the address

Reply:
Please recheck !